### PR TITLE
[ReactNative][Docs] Document how to link libraries

### PR DESCRIPTION
**Pull request by tadeuzagallo**
Add some documentation on how to link the libraries that we built as independent xcode projects.

/cc @vjeux @a2

### CODE REVIEW

File: docs/Libraries.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+### Step 2
+
+Click on your main project file (the one that represents the `.xcodeproj`)
+select `Build Phases` and drag the static library from the `Products` folder
+insed the Library you are importing to `Link Binary With Libraries`
+
+### Step 3
+
+The first two steps will be enough for all libraries that we ship we Ract Native
```
Comment (JoeStanton) [inline docs/Libraries.md]: s/we Ract Native/with React Native

**Comment by JoeStanton** [comment]
On the same sort of topic - I was going to have a stab at `react-native new-library` or similar, to generate a boilerplate XCode project to do Native bridging. Useful?

I found this a bit difficult/time consuming when writing RCTVibration with my limited XCode experience and it could encourage consistency and best practices around exports, docs etc.

**Comment by tadeuzagallo** [comment]
Oh, I'll fix this typo, thanks!

The `new-library` might be a good idea! @frantic, what do you think?

**Comment by frantic** [comment]
The article looks good, but maybe still pretty complex for non-ios developer who sees Xcode for the first time. Would be nice to add a screenshot. Ideally we would do all the work using something like `npm install <native-lib>`, but having docs is better than not having docs :)

As for `new-library` - good point, however I think the skills you need to build native library should already include some experience with Xcode. Still worth doing though.

**Pull request merged**